demisto_sdk/commands/common/git_util.py

```diff
@@ -102,6 +102,52 @@ def get_commit(self, commit_or_branch: str, from_remote: bool = True) -> Commit:
                     commit_or_branch, from_remote=from_remote, exception=e
                 )
 
+    def get_previous_commit(self, commit: Optional[str] = None) -> Commit:
+        """
+        Returns the previous commit of a specific commit or the head commit / branch.
```

Comment:
document what happens when `commit is None`

Reply:
done